src/networks/index.ts

```diff
@@ -68,6 +68,28 @@ export const networksByChainId: INetworkByChainId = {
     txExplorerUrl: 'https://testnet.teloscan.io/tx/',
     isMainnet: false,
   },
+  [ChainId.RSK]: {
+    name: 'RSK EVM Mainnet',
```

Comment:
just RSK Mainnet without "EVM" is ok. Also for testnet.
In Telos we use EVM because they have two network versions

Reply:
Done!